author/copy-from-rbenv.sh

```diff
@@ -1,45 +1,50 @@
 #!/bin/bash
-mkdir -p libexec completions
-rm -f completions/*
-rm -f \
-       libexec/plenv \
-       libexec/plenv---version \
-       libexec/plenv-commands \
-       libexec/plenv-completions \
-       libexec/plenv-exec \
-       libexec/plenv-global \
-       libexec/plenv-help \
-       libexec/plenv-hooks \
-       libexec/plenv-init \
-       libexec/plenv-local \
-       libexec/plenv-prefix \
-       libexec/plenv-rehash \
-       libexec/plenv-root \
-       libexec/plenv-sh-rehash \
-       libexec/plenv-sh-shell \
-       libexec/plenv-shims \
-       libexec/plenv-version \
-       libexec/plenv-version-file \
-       libexec/plenv-version-file-read \
-       libexec/plenv-version-file-write \
-       libexec/plenv-version-name \
-       libexec/plenv-version-origin \
-       libexec/plenv-versions \
-       libexec/plenv-whence \
-       libexec/plenv-which
-cp ~/.rbenv/libexec/* libexec/
-cp ~/.rbenv/completions/* completions/
+if [[ "$(ls -A ~/.rbenv)" && "$(ls -A ~/.rbenv/libexec)" && "$(ls -A ~/.rbenv/completions)" ]];
```

Comment:
Is it enough to use `-d` file test operator instead of `ls -A` ? `ls -A some_path` shows error if `some_path` does not exist like `ls: /Users/shohei.yoshida/.rbenv: No such file or directory`.

Reply:
I am fine with using -d if that's preferred.